client/app/components/onboardingGuide/GuideCoverPhotoPage.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import scss from './styles.scss';
+
+const GuideCoverPhotoPage = (props) => {
+
+  const handleClick = function(e, path) {
+    e.preventDefault();
+    props.changePage(path);
+  }
+
+  return (
+    <div className="container">
+      <a className={ scss.backLink }
+        onClick={ (e) => handleClick(e, "") }
+        href={ props.initialPath } >
+        ‹ { props.t('back_to_todo') }
```

Comment:
Passing in the translation helper function to child components ties them unnecessarily to the parent component. This seems like a sign of sharing functionality, but with means that is not the most intuitive one.